launch/docker.go

```diff
@@ -79,6 +82,10 @@ func (d *docker) runBuild(buildConfig buildConfig) error {
 		return err
 	}
 
+	err = execCommand("docker", "pull", buildImage).Run()
+	if err != nil {
+		return fmt.Errorf("failed to pull user image %v", err)
```

Comment:
same as above

Reply:
same as [this](https://github.com/screwdriver-cd/sd-local/pull/11/#discussion_r384979075)